examples/src/index.html

```diff
@@ -9,7 +9,7 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Roboto">
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./scss/style.scss">
```

Comment:
Just specify a source style file directly in `href` attribute. You can use a Webpack alias to source directory.
The html-bundler-webpack-plugin automatically exctract CSS into an output file and replaces the source filename with output filename